### PR TITLE
Add !intro to show a welcome message

### LINKED ISSUE #75: Add !welcome message
Add a welcome message that can be brought up with some short command. Simply to cut down on all the explaining we sometimes need to do for new members.

New guy: Hey I'm new here bla bla bla
Me: Welcome!
Me: !intro
Bot: Welcome to OTA! Make sure to check out our category specific channels #pwn, #web...

### PR DESCRIPTION
**Pull request by Kileak**
Fixes #75

### CODE REVIEW

**Comment by Grazfather** [comment]
💯

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ config.json
 *.bin
 *.log
 .venv
+intro_msg
```
Comment (Grazfather) [inline .gitignore]: newline error

File: intro_msg.template [hunk 2]
```diff
@@ -0,0 +1 @@
+Enter your welcome message here and copy this file to "intro_msg"
```
Comment (Grazfather) [inline intro_msg.template]: newline error

File: handlers/bot_handler.py [hunk 3]
```diff
@@ -18,11 +18,29 @@ def execute(self, slack_wrapper, args, channel_id, user_id):
...
+class IntroCommand(Command):
+    """Show an introduction message for new members."""
+
+    def execute(self, slack_wrapper, args, channel_id, user_id):
+        """Execute the Intro command."""
+        try:
+            with open("intro_msg") as f:
+                message = f.read()
+
+                slack_wrapper.post_message(channel_id, message)
```
Comment (Grazfather) [inline handlers/bot_handler.py]: nitpick: untab

File: handlers/bot_handler.py [hunk 4]
```diff
@@ -18,11 +18,29 @@ def execute(self, slack_wrapper, args, channel_id, user_id):
         """Announce the bot's presence in the channel."""
         slack_wrapper.post_message(channel_id, "Pong!")
 
+class IntroCommand(Command):
+    """Show an introduction message for new members."""
+
+    def execute(self, slack_wrapper, args, channel_id, user_id):
+        """Execute the Intro command."""
+        try:
+            with open("intro_msg") as f:
```
Comment (Grazfather) [inline handlers/bot_handler.py]: Tested within docker? works with cwd set by docker?

Comment (Kileak) [reply handlers/bot_handler.py]: Got me on this, only tested it from venv. But just checked, and it also finds it from within docker.

**Pull request merged**